### PR TITLE
Ret 800

### PR DESCRIPTION
**Pull request by MatthewDalbyV1**
[### JIRA link ###](https://tools.hmcts.net/jira/browse/RET-800)

added a button to return to claim which redirects you to login so you can continue with your existing claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/views/claim-saved.njk [hunk 1]
```diff
@@ -12,4 +12,9 @@
         {{ govukPanel({
         titleText: "Your claim has been saved"
       }) }}
+      {{ govukButton({
+        text: continueWithClaim,#
```
Comment (brendanmchugh) [inline src/main/views/claim-saved.njk:16]: Is the # a typo?

File: src/main/resources/locales/en/translation/claim-saved.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "continueWithClaim": "Continue with your claim"
```
Comment (brendanmchugh) [inline src/main/resources/locales/en/translation/claim-saved.json:2]: The 'Continue with your claim' could be moved into the json translation file also.

**Review by brendanmchugh — CHANGES_REQUESTED** [review]
Does this page display okay on preview? I think there might be some imports missing in the nunjucks file.

File: src/main/views/claim-saved.njk [hunk 3]
```diff
@@ -12,4 +13,9 @@
         {{ govukPanel({
         titleText: "Your claim has been saved"
       }) }}
+      {{ govukButton({
+        text: continueWithClaim,
+        href: "/return-to-existing"
```
Comment (pmumbere) [inline src/main/views/claim-saved.njk:18]: missing trailing comma

File: src/main/views/claim-saved.njk [hunk 4]
```diff
@@ -12,4 +13,9 @@
...
+        preventDoubleClick: true
+      })
```
Comment (pmumbere) [inline src/main/views/claim-saved.njk:20]: missing closing double braces

File: src/main/views/claim-saved.njk [hunk 5]
```diff
@@ -12,4 +13,9 @@
         {{ govukPanel({
         titleText: "Your claim has been saved"
       }) }}
+      {{ govukButton({
+        text: continueWithClaim,
```
Comment (pmumbere) [inline src/main/views/claim-saved.njk:17]: missing claim-saved.json translation file. You added continueWithClaim entry to home.json but do not reference home.json in the YourClaimHasBeenSavedController controller. I think you need to create translation file claim-saved.json and move the continueWithClaim field to it.

**Review by pmumbere — CHANGES_REQUESTED** [review]
Please address the missing trailing braces and comma issue. Also please introduce translation file corresponding to what is passed for rendering in your controller. Also from the mock-up on the JIRA ticket I can see you are missing a paragraph with content

**Review by pmumbere — APPROVED** [review]

**Review by brendanmchugh — APPROVED** [review]

**Pull request merged**